src/promptflow/promptflow/errors.py

```diff
@@ -0,0 +1,26 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from promptflow.exceptions import UserErrorException, ValidationException
+
+
+class ValueErrorException(ValidationException, ValueError):
+    pass
+
+
+class NotImplementedErrorException(UserErrorException, NotImplementedError):
```

Comment:
If user need to take action, we should categorize as UserError. If pf team needs to take action, it's non-user-error

Reply:
ok.